README.md

```diff
@@ -148,3 +148,200 @@ See [LICENSE](./LICENSE) for more details.
+	
+		api-subnet-route-table
+		- keep destination 0.0.0.0/0
+		pods-subnet-route-table
+		- remove all destinations except 0.0.0.0/0
+		services-subnet-route-table
+		- keep destination 0.0.0.0/0
+		workers-subnet-route-table
+		- remove all destinations except 0.0.0.0/0
+		
+		api-nsg
+		-  keep ingress
+		      api-nsg (TCP/6443)
+		      pods-nsg (TCP/6443)
+		      pods-nsg (TCP/12250)
+		      workers-nsg (TCP/6443)
+		      workers-nsg (TCP/10250)
+		      workers-nsg (TCP/12250)
+		      workers-nsg (ICMP)
+		      bastion 10.3.48.0/20 (TCP/6443)
+		-  remove egress SGW
+		-  keep egress
+		      api-nsg (TCP/6443)
+		      pods-nsg (All/All)
+		      workers-nsg (TCP/10250)
+		      workers-nsg (TCP/12250)
+		      workers-nsg (ICMP)
+		pods-nsg
+		-  remove all ingress except
+		      api-nsg (All/All)
+		      pods-nsg (All/All)
+		      workers-nsg (All/All)
+		-  remove all egress except
+		      api-nsg (TCP/6443)
+		      api-nsg (TCP/12250)
+		      pods-nsg (All/All)
+		      OPTIONAL: 0.0.0.0/0 (TCP/All)
+		services-nsg
+		-  keep ingress except 0.0.0.0/0 (All/All)
+		-  remove all egress except
+		      workers-nsg (TCP/10256)
+		      workers-nsg (TCP/30000-32767)
+		      workers-nsg (ICMP)
+		workers-nsg
+		-  remove all ingress except
+		      api-nsg (All/All)
+		      services-nsg (TCP/10256)
+		      services-nsg (TCP/30000-32767)
+		      workers-nsg (All/All)
+		      bastion 10.3.3.0/28 (TCP/22)
+		      0.0.0.0/0 (ICMP)
+		-  remove all egress except
+		      api-nsg (TCP/6443)
+		      api-nsg (TCP/10250)
+		      api-nsg (TCP/12250)
+		      pods-nsg (All/All)
+		      workers-nsg (All/All)
+		      0.0.0.0/0 (ICMP)
+		      OPTIONAL: 0.0.0.0/0 (TCP/All)
+	
+	
+	Hub and Spoke Routing VCN (zt-hub-vcn):
+	
+		web-subnet-route-table
+		- keep IGW
+		outdoor-subnet-route-table
+		- keep NGW
+		- keep SGW
+		indoor-subnet-route-table
+		- add 10.0.0.0/20, 10.1.0.0/20, 10.2.0.0/20, 10.3.0.0./16
+		- keep SGW
+		mgmt-subnet-route-table
+		- keep NGW
+		- keep SGW
+	
+		outdoor-nlb-nsg
+		- keep ingress 0.0.0.0/0 (TCP/443)
+		- keep egress outdoor-fw-nsg (TCP/All)
+		outdoor-fw-nsg
+		- keep ingress outdoor-nlb-nsg (TCP/80)
+		- keep egress 0.0.0.0/0 (TCP/All)
+		indoor-fw-nsg
+		- keep ingress indoor-nlb-nsg (TCP/80)
+		- keep egress 0.0.0.0/0 (TCP/All)
+		indoor-nlb-nsg
+		- keep ingress 0.0.0.0/0 (TCP/80)
+		- keep egress indoor-fw-nsg (TCP/All)
+		jump-host-nsg
+		- no ingress
+		- keep egress mgmt-nsg (TCP/22)
+		mgmt-nsg
+		- keep ingress
+		      jump-host-nsg (TCP/22)
+		      bastion service 192.168.0.48/28 (TCP/22)
+		      bastion service 192.168.0.48/28 (TCP/443)
+		- no egress
+
+
+### On-Premises Connectivity
+
+* Configuration of Fast Connnect, routing, NSGs, etc. for specific customer on-premises facilities is beyond the scpoe of Zero Trust Landing Zone.  Customer connctivity needs to be added manually, as a post-deployment exercize.
+
+### Identity Domain bug
+
+* Details???
```

Comment:
do we want to add any content here?

Reply:
Yes, Josh is gathering details/verifying the bug.